### ISSUE TITLE
Collapsable Icon Menu

### ISSUE THREAD

**Issue by henokrb** [zuramai/mazer#33]
Hi,

Can you add  Collapsable Responsive Menu with only Icons Showing? 

Submenus will be shown when one hoovers on the icons.

Thank you

**Comment by jose-jimmy** [comment]
Hi @fzn0x can I work on this issue.

**Comment by fzn0x** [comment]
> Hi @fzn0x can I work on this issue.

Ofcourse, I will assign this issue to you,

**Comment by fzn0x** [comment]
Assigned, PR welcome 😃

**Comment by jose-jimmy** [comment]
Working on it 👍🏻 , just to make sure we are on the same page,
 ◀️ so I have to make the menu bar in such a way that it shows only the icons 
 ◀️ when hovered over the icons it should display all the sub menus on top of all other layers

**Comment by bilalshaikh292** [comment]
Heyy I has been a long time and the issue is not not yet solved ? I would like to contribute?

**Comment by samhk90** [comment]
hey can u assign me this issue

**Comment by SurajPrakash24** [comment]
Hey, Can You plz assign it to me. I would like to contribute to this feature

**Comment by alfianchii** [comment]
Certainly, all of you are welcome to initiate a PR @bilalshaikh292 @samhk90 @SurajPrakash24. Following that, I would be pleased to allocate this issue to those who have created the PR for it.

**Comment by bilalshaikh292** [comment]
![image](https://github.com/zuramai/mazer/assets/110531971/ef0630fc-f1be-45dc-a82f-484817ede47d)
Heyy @alfianchii Something like this would work ? Let me Know ?

**Comment by alfianchii** [comment]
> ![image](https://github.com/zuramai/mazer/assets/110531971/ef0630fc-f1be-45dc-a82f-484817ede47d)
> Heyy @alfianchii Something like this would work ? Let me Know ?

Yup.
